src/pages/resource/contributors/StyledComponents.tsx

```diff
@@ -9,15 +9,29 @@ import DeleteIcon from '@material-ui/icons/Delete';
 import Person from '@material-ui/icons/Person';
 import AddCircleIcon from '@material-ui/icons/AddCircle';
 
+const BaseSelectStyle = styled(Select)`
+  height: 2rem;
+  background-color: white;
```

Comment:
theme? kan kanskje settes for alle Select-komponenter via `mainTheme.js` også?

Reply:
bør iallefall bruke theme-farge-paletten